src/game/client/components/players.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include <engine/textrender.h>
 
+#include "base/color.h"
```

Comment:
Conventionally we do absolute imports with <>.
```suggestion
#include <base/color.h>
```

Reply:
oh, clangd did that, didnt even notice xd, but will change